core/app/views/layouts/refinery/admin.html.erb

```diff
@@ -2,22 +2,30 @@
 <%= render '/refinery/html_tag' %>
   <% content_for :meta, tag(:meta, name: 'refinerycms', content: Refinery.version) %>
   <%= render 'refinery/admin/head' %>
-  <body class="<%= action_name %> <%= I18n.locale %>">
+  <body class="<%= action_name %> <%= I18n.locale %> cms-interface">
     <%= render 'refinery/admin/menu' %>
     <%= render '/refinery/site_bar' %>
     <div id='tooltip_container'></div>
-    <div id="admin_container" class="clearfix">
-      <div id="page_container">
-        <div id="page">
-          <div id="content" class="clearfix">
-            <div id="flash_container">
-              <%= render '/refinery/no_script' %>
-              <%= render '/refinery/message' %>
+    <section id="wrapper" class="dimmer">
+      <% if content_for?(:admin_content) %>
```

Comment:
> How's that? Good to go? Do you prefer to have separate commits as it is in progress so the commits match the comments and conversation (and then squash at the end when you say to)? Or shall I just squash everything as I have updates?

I've spoke about to clean commits log of the PR. Not squashing all commits into one. Sorry for the misunderstood

For example, if the first commit add features 1-2-3 and the second commit remove the feature 3, you can squash the commit to have only 1 commit with features 1-2.